standards/storing-secrets.md

```diff
@@ -0,0 +1,25 @@
+---
+category: Building software
+expires: 2019-01-01
+---
+# Storing secrets
+
+All our secrets should be adequately protected and suitably stored.
+
+Where possible, use infrastructure-based secrets management services such as [AWS Key Management Service](https://aws.amazon.com/kms/), [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html), [Microsoft Azure Key Vault](https://azure.microsoft.com/en-gb/services/key-vault/) or [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) on MOJ's Cloud Platforms.
```

Comment:
Maybe link to the Cloud Platform docs for secrets (`https://ministryofjustice.github.io/cloud-platform-user-docs/02-deploying-an-app/003-add-secrets-to-deployment/`) instead of generic k8s docs?

I might also suggest rewriting this to be more directive, like:

```markdown

Use infrastructure-based secrets management services where possible.

- On the MOJ Cloud Platform, use Kubernetes secrets
- On AWS, use KMS
- On Azure, use Key Vault
```

Reply:
Changed